fastembed/text/onnx_embedding.py

```diff
@@ -229,7 +229,8 @@ def __init__(
 
         model_description = self._get_model_description(model_name)
         cache_dir = define_cache_dir(cache_dir)
-        model_dir = self.download_model(model_description, cache_dir)
+        local_files_only = kwargs.get('local_files_only', False)
```

Comment:
if we've already extracted local_files_only into `self.local_files_only` in `TextEmbeddingBase`, why do we get it from `kwargs` again and not using `self.local_files_only`?